src/Html/HtmlCssSelectors.fs

```diff
@@ -51,6 +51,14 @@ module internal HtmlCssSelectors =
         let getOffset (t:List<char>) = 
             charCount - 1 - t.Length
 
+        let isCharacterEscapable (c:char) =
+            (* CSS 2.1: Any character (except a hexadecimal digit, linefeed,
+               carriage return, or form feed) can be escaped with a backslash to
+               remove its special meaning *)
+            let isHexadecimalDigit = Char.IsDigit(c) || (Char.ToLower(c) >= 'a' && Char.ToLower(c) <= 'f')
+            (isHexadecimalDigit || c = '\n' || c = '\f' || c = '\r')
```

Comment:
Sorry, yes intended to say I'd prefer the not upfront in this specific case.

Yes both can be used liberally, when I use it with pipe in simple expressions though it is most often a sign of my own lazyness (and lack of skills with editor) because I can just append it instead of putting paren back.

I like it used in chain of pipes.